src/h3lib/lib/coordijk.c

```diff
@@ -354,8 +349,8 @@ void _downAp7r(CoordIJK* ijk) {
  * @param ijk The ijk coordinates.
  * @param digit The digit direction from the original ijk coordinates.
  */
-void _neighbor(CoordIJK* ijk, int digit) {
-    if (digit != 0) {
+void _neighbor(CoordIJK* ijk, Direction digit) {
```

Comment:
Nit: Should this argument be `dir` now? It seems like we're a little inconsistent still about whether we call this a "direction" (the semantic name) or a "digit" (the concrete implementation).

Reply:
Even worse, the directions are themselves called "digits"